code/modules/cargo/exports/manifest.dm

```diff
@@ -76,19 +76,3 @@
 /datum/export/manifest_correct_denied/get_cost(obj/O)
 	var/obj/item/paper/fluff/jobs/cargo/manifest/M = O
 	return ..() - M.order_cost
-
-// Paper work done correctly
-
-/datum/export/paperwork_correct
```

Comment:
Dont remove this bit, I plan on making it into a fax thing.
Were you add paper to a fax-er and get paper work back to complete

Reply:
You do your revamp, meanwhile I'm taking out the trash. It's not even 50 lines long.